moped-editor/src/views/projects/projectView/ProjectFundingTable.js

```diff
@@ -152,7 +167,7 @@ const ProjectFundingTable = () => {
 
   const { loading, error, data, refetch } = useQuery(FUNDING_QUERY, {
     // sending a null projectId will cause a graphql error
-    // id 0 used when creating a new project, no project personnel will be returned
+    // id 0 used when creating a new project, no project funding will be returned
     variables: {
       projectId: projectId ?? 0,
     },
```

Comment:
Please correct me if I'm misunderstanding, but I am under the impression that a user should never be at this part of an app without there being a `projectId` defined and available to provide to the query. 

If that's the case, and if a user somehow does get to this page without a particular project (I really don't think this is a possible route), I would suggest we issue a redirect or somehow send them back to the project listing and not try to render anything with inevitably null or blank data.

I think this query *should* break if there isn't a projectId, and putting a `0` in there as a default value opens up the possibility for the app to think we're dealing with project `0` and have potential knock on effects. 

That said, I don't think this is a big deal, just a thought to consider.

Reply:
You're absolutely right. This is a hold over from when the add project flow contained more tables than just the title and description. The same pattern is on the personnel table. 

If I'm remembering correctly, if a null projectId was in the query, it would fail with an error, but sending 0 would return empty data.